packages/firestore/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/firestore/classes/options/AddDocumentSnapshotListenerOptions.java

```diff
@@ -1,19 +1,31 @@
 package io.capawesome.capacitorjs.plugins.firebase.firestore.classes.options;
 
+import androidx.annotation.Nullable;
+
 public class AddDocumentSnapshotListenerOptions {
 
     private String reference;
+    private final boolean includeMetadataChanges;
     private String callbackId;
 
-    public AddDocumentSnapshotListenerOptions(String reference, String callbackId) {
+    public AddDocumentSnapshotListenerOptions(
+            String reference,
+            @Nullable Boolean includeMetadataChanges,
+            String callbackId
+    ) {
         this.reference = reference;
+        this.includeMetadataChanges = includeMetadataChanges != null ? includeMetadataChanges : false;
```

Comment:
```suggestion
        this.includeMetadataChanges = includeMetadataChanges == null ? false : includeMetadataChanges;
```

Keep it simple.

Reply:
I guess it's a matter of taste. I prefer to keep the default at the end, analogous to the JS syntax `includeMetadataChanges ?? false`. However, it's your project, so I'll stick to your coding style. 😊

Reply:
You're right. It really is a matter of taste. Thank you for changing it anyway.